code/style.css

```diff
@@ -9,50 +11,141 @@ p {
   background-color: #fff;
 }
 
+.header-styling {
+  color: rgba(10, 11, 13, 0.4);
+  text-shadow: 2px 4px 4px white;
+  text-align: right;
+  background: rgba(10, 11, 13, 0.7);
+  height: 100px;
+  font-size: 70px;
+  padding: 30px;
+}
+
 .cards {
   margin: 50px auto;
   width: 90%;
   height: 600px;
-  background: #ccc;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  border-radius: 20px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.2);
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: rgba(10, 11, 13, 0.7);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transition: transform 0.2s;
+}
+
+.killer-deck:hover {
+  transform: scale(1.1);
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: rgba(10, 11, 13, 0.7);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transition: transform 0.2s;
+}
+
+.weapon-deck:hover {
+  transform: scale(1.1);
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: rgba(10, 11, 13, 0.7);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transition: transform 0.2s;
+}
+
+.room-deck:hover {
```

Comment:
This property could have been put together with killer-deck:hover and weapon-deck hover since they do the exact same thing.

Reply:
It also seems like the `.room-deck` and `.weapon-deck` and `.killer-deck` have similar CSS properties